### PR TITLE
UsersManager created and working

### PR DESCRIPTION
**Pull request by Borjarguez**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<IfModule mod_rewrite.c>
+    RewriteEngine on
+    RewriteRule    ^$    webroot/    [L]
+    RewriteRule    (.*) webroot/$1    [L]
+	
+	Header set Access-Control-Allow-Methods "PUT, GET, POST, DELETE, OPTIONS"
```
Comment (LeeScoresby) [inline .htaccess:6]: Indentation review

File: .vscode/sftp.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Colmena Project",
```
Comment (LeeScoresby) [inline .vscode/sftp.json:2]: COLMENA Manager is the true name

**Review by LeeScoresby — CHANGES_REQUESTED** [review]
Pending actions to be done:

- Make sure that vendors are not included in the repo
- Avoid to add files like .lock and similar, who depends on the local environment
- Delete every unnecesary config default file
- Prepare a gitignore which avoids include in the repo the vendors, minified and so on

The idea is to clone the repo, make a composer install (and the same in /webroot) and update dependencies remotely, not because they're included as files in the repo.

File: .vscode/sftp.json (hunk 2 above)
Comment (Borjarguez) [reply .vscode/sftp.json:2]: I consider that vscode files should not be included in the repo, so i added it to gitignore in one of the more recent commits

**Review by Borjarguez — COMMENTED** [review]
I think everything that can be deleted from the repo are now at gitignore and deleted

**Review by LeeScoresby — APPROVED** [review]

**Pull request merged**